Assets/Helpers/bevy_resolution.toml

```diff
@@ -0,0 +1,4 @@
+name = "bevy_resolution"
+description = "Library which aims to make resolutions easier and simplier to work with."
```

Comment:
```suggestion
description = "Library which aims to make resolutions easier and simpler to work with."
```

Reply:
I somehow didn't notice that. Thanks for catching that!